packages/orchestration/src/exos/local-chain-account-kit.js

```diff
@@ -44,7 +44,12 @@ const { Fail } = assert;
 const HolderI = M.interface('holder', {
   getPublicTopics: M.call().returns(TopicsRecordShape),
   makeDelegateInvitation: M.call(M.string(), AmountShape).returns(M.promise()),
+  makeUndelegateInvitation: M.call(M.string(), AmountShape).returns(
+    M.promise(),
+  ),
```

Comment:
did you consider `M.callWhen(...).returns(InvitationShape)`?

Reply:
I think I just copied this one. good idea tho